.eslintrc

```diff
@@ -16,7 +16,7 @@
     "no-console": "off",
     "no-extra-boolean-cast": "off",
     "no-undef": "error",
-    "no-unused-vars": "off",
+    "no-unused-vars": "error",
```

Comment:
The extension `eslint:recommended` has this rule already enabled, I think we should now get rid of it here